### DISCUSSION TITLE
How to display each images per row dynamically

### DISCUSSION THREAD

**Discussion by gitalvininfo** [exceljs/exceljs#2476] [Q&A]
So I have an array of images and I'm using base64 to display images.

`const images = [{name: "dog", link: base64link}, {name: "cat", link: base64link}, {name: "bird", link: base64link}]`

Can I display this images per row without specifying the fix cell position of this images.  I just want to display it vertically and continuously to bottom depending of the number of data inside my array

       A         B
1   Dog    Image
2   Cat     Image
3   Bird    Image

My array is not fix so the data is somewhat changing over time. Please let me know if this is possible. Thanks in advance

**Comment by HansvdLaan** [comment]
+1 for this question
